validation-test/Sema/type_checker_crashers_fixed/rdar27017206.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: not --crash %target-swift-frontend %s -typecheck
+// RUN: not %target-swift-frontend %s -typecheck
 // REQUIRES: asserts
```

Comment:
Please drop the "REQUIRES: asserts" as well. (We're not good about this and often clean them up in large batches, but it's good to get in the habit.)